boards/arm/bl5340_dvk/bl5340_dvk_cpunet_reset.c

```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright (c) 2019-2021 Nordic Semiconductor ASA.
+ * Copyright (c) 2021 Laird Connectivity
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr.h>
+#include <init.h>
+#include <logging/log.h>
+
+#include <soc.h>
+
+LOG_MODULE_REGISTER(bl5340_dvk_cpuapp, CONFIG_LOG_DEFAULT_LEVEL);
+
+#if !defined(CONFIG_TRUSTED_EXECUTION_NONSECURE)
+
+/* This should come from DTS, possibly an overlay. */
+#if defined(CONFIG_BOARD_BL5340_DVK_CPUAPP)
+#define CPUNET_UARTE_PIN_TX  8
+#define CPUNET_UARTE_PIN_RX  10
+#define CPUNET_UARTE_PORT_TRX NRF_P1
+#define CPUNET_UARTE_PIN_RTS 7
+#define CPUNET_UARTE_PIN_CTS 9
+#define CPUNET_UARTE_PORT_RCTS NRF_P1
+#endif
+
+#if defined(CONFIG_BT_CTLR_DEBUG_PINS_CPUAPP)
+#include <../subsys/bluetooth/controller/ll_sw/nordic/hal/nrf5/debug.h>
+#else
+#define DEBUG_SETUP()
+#endif
```

Comment:
`BT_CTLR_DEBUG_PINS_CPUAPP` is an option only available for nRF5340 DK, see:
https://github.com/zephyrproject-rtos/zephyr/blob/0923e32fe743e71103ac660a50fd6fb2c5968ef6/subsys/bluetooth/controller/Kconfig#L625-L627
so I think this can be removed (together with the call to `DEBUG_SETUP()` in [line 50](https://github.com/zephyrproject-rtos/zephyr/pull/34833/files#diff-04678edb346d8cf445ddeb02e3394986d59514b695884e60c594187c48ac8eefR50)).